tests/modules/asset_groups/resources/test_permissions.py

```diff
@@ -107,10 +105,7 @@ def test_create_patch_asset_group(
         asset_group_utils.delete_asset_group(
             flask_app_client, researcher_1, asset_group_guid
         )
-        try:
-            delete_remote(str(temp_asset_group.guid))
-        except GitlabInitializationError:
-            pass
+        delete_remote(str(temp_asset_group.guid))
```

Comment:
Surely if you have delete the asset group this shouldn't be necessary. In fact  the real test would be to ensure that it's not on the remote.